### PR TITLE
ADCM-6195: Add supported ADCM version in client

### PR DESCRIPTION
**Pull request by DanBalalan**

### CODE REVIEW

File: adcm_aio_client/core/client/_client.py [hunk 1]
```diff
@@ -50,10 +60,32 @@ async def build_client(
...
     await requester.login(credentials=Credentials(username="admin", password="admin"))  # noqa: S106
-    return ADCMClient(requester=requester, bundle_retriever=BundleRetriever())
+    return ADCMClient(requester=requester, bundle_retriever=BundleRetriever(), adcm_version=adcm_version)
+
+
+async def _get_and_check_adcm_version(url: str, timeout: float) -> str:
+    try:
+        adcm_version = await _get_adcm_version(url=url, timeout=timeout)
+    except VersionRetrievalError as e:
+        message = f"Can't get ADCM version for {url}"
```
Comment (Sealwing) [inline adcm_aio_client/core/client/_client.py:77]: let's specify that it's more likely due to adcm ver < 2.5.0

File: tests/integration/test_dummy.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from adcm_aio_client.core.client import ADCMClient
+from adcm_aio_client.core.client._client import ADCMClient
```
Comment (Sealwing) [inline tests/integration/test_dummy.py:5]: why introducing _client?

Comment (DanBalalan) [reply tests/integration/test_dummy.py:5]: To make clear, that `build_client` is the only convenient way to get an ADCMClient

File: adcm_aio_client/core/client/_client.py (hunk 1 above)
Comment (DanBalalan) [reply adcm_aio_client/core/client/_client.py]: Added line to error

File: tests/integration/test_dummy.py (hunk 2 above)
Comment (DanBalalan) [reply tests/integration/test_dummy.py]: returned core/client.py

**Review by Sealwing — APPROVED** [review]

**Review by Starovoitov — DISMISSED** [review]

**Review by Sealwing — APPROVED** [review]

**Review by Starovoitov — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Pull request merged**